leaderboard/views.py

```diff
@@ -427,7 +427,17 @@ def teampage(request):
         data_triples.append((key, primary[key], data[key]))
 
     # Details needed for the post-submission/publication survey
-    publication_survey = {'active': False, 'team': current_team}
+    publication_survey = {
+        'active': True,
+        'username': '{} ({} from OCELoT)'.format(
+            current_team.institution_name, current_team.name
```

Comment:
Note that I used `current_team.name` instead of `current_team.email` as I was not sure if we can export e-mails from OCELoT for privacy reasons, etc.